src/omotes_simulator_core/adapter/transforms/mappers.py

```diff
@@ -1,19 +1,5 @@
-#  Copyright (c) 2023. Deltares & TNO
-#
-#  This program is free software: you can redistribute it and/or modify
-#  it under the terms of the GNU General Public License as published by
-#  the Free Software Foundation, either version 3 of the License, or
```

Comment:
Why did you remove the copyright statement?

Reply:
it was by mistake, i put it back.